src/components/lists/lists.module.scss

```diff
@@ -0,0 +1,78 @@
+@import 'src/assets/stylesheets/base_styles/colors';
+@import 'src/assets/stylesheets/utilities/functions';
+@import 'src/assets/stylesheets/base_styles/typography';
+@import 'src/assets/stylesheets/base_styles/break_points';
+
+$article-list-color:                 $neutral-2;
+$article-list-heading-color:         $neutral-4;
+$article-list-heading-hover-color:   $neutral-1;
+$articles-list-heading-border-color: $action-2;
+$article-list-image-max-width:       ru(10);
+$articles-list-tag-color:            $neutral-3;
+
+.article-list {
+  list-style: none;
+  color: $article-list-color;
+}
+
+li {
+  margin-bottom: ru(1);
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.header {
+  @include like-h5;
```

Comment:
I'm ripping this `like-h5` mixin out in my PR. IMO we shouldn't be using it, we should just be using the typography mixins.